test/docfx.Test/e2e/E2ETest.cs

```diff
@@ -32,7 +32,7 @@ public static class E2ETest
             "download-failed", "locale-invalid",
 
             // show multiple errors with line info
-            "publish-url-conflict", "output-path-conflict", "uid-conflict", "redirection-conflict",
+            "publish-url-conflict", "output-path-conflict", "uid-conflict", "uid-name-conflict", "redirection-conflict",
```

Comment:
we should be able to report where the `uid` is defined, but it could be done in another PR.

Reply:
We should fix the errors of conflicting at some point.